backend/oracleobjectstorage/object.go

```diff
@@ -363,7 +363,14 @@ func (o *Object) Open(ctx context.Context, options ...fs.OpenOption) (io.ReadClo
 	if err != nil {
 		return nil, err
 	}
-	o.bytes = *bytes
+	if bytes == nil {
+		// Must set these as "defaults" since the HTTP payload is compressed and the handler will decompress by default,
```

Comment:
please surface the gzip test results. 

Also could we please copy the s3 Open function way to keep the code in same pattern for handling gzip stuff. I saw related code in 

https://github.com/rclone/rclone/blob/master/backend/s3/s3.go#L5493
https://github.com/rclone/rclone/blob/master/backend/s3/s3.go#L5550

please search for gzip in s3.go and compare it:
https://github.com/search?q=repo%3Arclone%2Frclone+path%3A%2F%5Ebackend%5C%2Fs3%5C%2F%2F+zip&type=code